public/home.html

```diff
@@ -26,6 +26,7 @@
   <script type="text/javascript" src="js/extraFunctions.js"></script>
   <script type="text/javascript" src="js/buttonFunctions.js"></script>
   <link rel="stylesheet" type="text/css" href="fonts/fontawesome/css/all.min.css">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```

Comment:
This should be included via `node_modules` and exposed to the frontend via `serve.js`. I can take care of this; I am just noting this here as a to-do item.